security/hosts_multitenancy.adoc

```diff
@@ -8,68 +8,50 @@
+In Linux, containers are just a special type of process, so securing containers
+is the same as securing any running process. Containers should run as a non-root
+user. Dropping the privilege level or creating containers with the least amount
+of privileges possible is recommended.
 
-[[security-hosts-how-containers-are-secured-on-rhel]]
-== How Containers are Secured on RHEL
+The following security features are available to secure containers running on
+RHEL and RHEL Atomic Host:
 
-In Linux, containers are really just a special type of process, so your approach
-to securing containers should be the same as your approach to securing any
-running process. Dropping privileges is important and still recommended, though
-even better is to create containers with the least privileges possible.
-Containers should run as a non-root user.
-
-There are multiple levels of security features available for securing containers
-running on RHEL and RHEL Atomic Host:
-
-- _Linux namespaces_ provide the fundamentals of container isolation. A namespace
-makes it appear to the processes within the namespace that they have their own
-instance of global resources. Namespaces provide the abstraction that gives the
-impression you are running on your own operating system when you are inside a
-container.
+- _Linux namespaces_ enable creating an abstraction of a particular global system
+resource to make it appear as a separate instance to processes within a
+namespace. Consequently, several containers can use the same resource
+simultaneously without creating a conflict.
 
 - _SELinux_ provides an additional layer of security to keep containers isolated
 from each other and from the host. SELinux allows administrators to enforce
 mandatory access controls (MAC) for every user, application, process, and file.
-SELinux is the brick wall that is going to stop you if you manage to break out
-of (accidentally or on purpose) the namespace abstraction.
 
-- _CGroups_ (or control groups) limit, account for, and isolate the resource usage
+- _CGroups_ (control groups) limit, account for, and isolate the resource usage
 (CPU, memory, disk I/O, network, etc.) of a collection of processes. CGroups are
-used to ensure your container will not impacted by another container on the same
-host.
+used to ensure that containers on the same host are not impacted by each other.
 
-- Finally, a _secure computing mode (seccomp)_ profile can be associated with a
-container to restrict available system calls.
+- _Secure computing mode (seccomp)_ profiles can be associated with a container to
+restrict available system calls.
 
-You can further enhance security for your applications and infrastructure by
+You can also enhance security for your applications and infrastructure by
 deploying your containers to RHEL Atomic Host. RHEL Atomic Host reduces the attack
 surface by minimizing the host environment and tuning it for containers.
```

Comment:
We already say above that the reader can use RHEL Atomic. Maybe only the second sentence is necessary here, and perhaps even moved up before the features to fit with more of the "OSs you can use" vibe, as this is mentioning a little of the 'why' when describing the differences between this and vanilla RHEL.